### DISCUSSION TITLE
Telekasten for Fzf-lua?

### DISCUSSION THREAD

**Discussion by vinoff** [ibhagwan/fzf-lua#1224] [General]
There is this awesome plugin called Telekasten: https://github.com/nvim-telekasten/telekasten.nvim

It is a "plugin for working with a markdown zettelkasten / wiki and mixing it with a journal, based on telescope.nvim". It is honestly perfect. It uses Telescope though.

Do you guys know of any alternative that uses Fzf-lua instead? I am in the process of moving from Telescope to Fzf-lua.

**Comment by ibhagwan** [comment]
I don’t use a notes app so I can’t recommend based on personal usage but I know Obsidian supports fzf-lua:
https://github.com/epwalsh/obsidian.nvim

If you really like Telekasten you can PR it for fzf-lua support, [the API](https://github.com/ibhagwan/fzf-lua/wiki/Advanced) is very simple and easy to use.
